ipa-core/src/protocol/basics/shard_fin.rs

```diff
@@ -288,9 +288,7 @@ where
         C: 'a,
     {
         async move {
-            // todo: SixteenBit only works for values up to BA16. EightBitStep will panic if we try
-            // to add larger values
-            self.values = integer_sat_add::<_, SixteenBitStep, B>(
+            self.values = integer_sat_add::<_, ThirtyTwoBitStep, B>(
```

Comment:
does this mean we should use BA32 instead of BA16 for the HV generic?

Reply:
no, this is just to make compact gate stuff work